src/modules/products/services/category.service.ts

```
@@ -0,0 +1,94 @@
+      return await this.categoryRepository.save(category);
+    } catch (error) {
+      throw new BadRequestException('Failed to create category');
+    }
+  }
+
+  async update(
+    id: string,
+    updateCategoryDTO: UpdateCategoryDTO,
+  ): Promise<Category> {
+    try {
+    } catch (error) {}
```

ESLint

[error] 'error' is defined but never used.

(@typescript-eslint/no-unused-vars)